automation/test.sh

```diff
@@ -280,9 +280,11 @@ elif [[ $TARGET =~ genie.* ]]; then
 elif [[ $TARGET =~ sriov.* ]]; then
   ginko_params="$ginko_params --ginkgo.focus=SRIOV"
 elif [[ $TARGET =~ gpu.* ]]; then
-  ginko_params="$ginko_params --ginkgo.focus=GPU" 
+  ginko_params="$ginko_params --ginkgo.focus=GPU"
 elif [[ $TARGET =~ (okd|ocp).* ]]; then
   ginko_params="$ginko_params --ginkgo.skip=Genie|SRIOV|GPU"
+elif [[ $TARGET =~ ipv6.* ]]; then
+  ginko_params="$ginko_params --ginkgo.skip=Multus|Genie|SRIOV|GPU|.*slirp.*|.*bridge.*|.*PVC.*"
```

Comment:
PVC is skipped until https://github.com/kubevirt/kubevirt/pull/3171 is merged.
I think I can remove the Multus, SRIOV and GPU skip.
@qinqon is there a spacial reason you skipped it? Are you ok with removing it?

Reply:
I'm afraid we cannot test this with Multus, nor SR-IOV. kindnet (which provides us with IPv6 support) does not play well with Multus.

While we were developing kind-sriov lane, we were not able to use kindnet+multus, so we switched to flannel and later calico.

So since we need kindnet for IPv6 and kindnet does not work with Multus, we cannot enabled none of these two lanes.

We may try again enabling Multus on kind to see if anything changed since, but we cannot use the current provider.

I don't see this as that big of a risk since we connect Multus interfaces only through `bridge` binding and SR-IOV through `sriov`.

Reply:
@rmohr before I'm adding it back. What is GPU? Does it supported on kind provider?

Reply:
What I did was just copy the default lane and skip slirp and bridge, about multus, we can try to hack calico into kind provider but noone has do it, let's wait to have a proper provider the k8s-1.17 has already calico running and the k8s-1.17 is passing most of the times here, also I think Roman is preparing the dnsmasq container to assign ipv6 addresses too.

Reply:
btw there are more PVC related test without PVC in the test name